src/jats/__tests__/data/section-categories.ts

```diff
@@ -0,0 +1,227 @@
+/*!
+ * © 2024 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SectionCategory } from '../../../schema'
+
+export const sectionCategories: SectionCategory[] = [
+  {
+    id: 'abbreviations',
+    titles: ['Abbreviations'],
+    synonyms: ['list of abbreviations', 'abbreviations'],
+    group: 'body',
+    isUnique: false,
+  },
+  {
+    id: 'abstract',
+    titles: ['Abstract'],
+    synonyms: ['abstract'],
+    group: 'abstracts',
+    isUnique: false,
+  },
+  {
+    id: 'abstract-graphical',
```

Comment:
We should not need this since this has its own node.

Reply:
Removed.